pkgs/development/interpreters/starlark/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+buildGoModule rec {
+  pname = "starlark";
+  version = "unstable-2022-02-13";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "starlark-go";
+    rev = "c55a923347b1ca17cb609b1570187075c6bb911f";
+    sha256 = "sha256-Str7PKQqbI0SniRJDompwlshHmC299EZzqEb4pYsd+8=";
+  };
+
+  vendorSha256 = "sha256-lgL5o3MQfZekZ++BNESwV0LeoTxwEZfziQAe99zm4RY=";
```

```suggestion
  version = "unstable-2022-03-02";

  src = fetchFromGitHub {
    owner = "google";
    repo = "starlark-go";
    rev = "5411bad688d12781515a91cc032645331b4fc302";
    sha256 = "sha256-JNsGyGlIVMS5w0W4jHVsrPqqNms3Xfpa4n/XcEWqt6I=";
  };

  vendorSha256 = "sha256-lgL5o3MQfZekZ++BNESwV0LeoTxwEZfziQAe99zm4RY=";
```
This should be an update to the latest version.